doc/en/user/source/community/smart-data-loader/data-store.rst

```diff
@@ -6,46 +6,46 @@ Smart Data Loader
 Data Store 
 ----------
 
-In addition to the parameters common to each ``DataStore`` configuration as ``workspace``, ``name`` and ``description``, the page shows the following connection parameters in the dedicated section:
+In addition to the parameters common to each ``DataStore`` configuration such as ``workspace``, ``name`` and ``description``, the page shows the following connection parameters in the dedicated section:
 
-* ``Data store name``: the name of the PostGIS ``DataStore`` already created under the same workspace. It will be used to access the database and build xsd and mappings file based on the databse metadata.
+* ``Data store name``: the name of the PostGIS ``DataStore`` already created under the same workspace. It will be used to access the database and build xsd and mappings file based on the database metadata.
 
 * ``Root entity``: the name of the DB table to be used as the root FeatureType to build the xsd and the mapping file.
 
-Once selected the root entity a diagram of the entities at stake will appear, allowing you to uncheck the attributes that are not meant to be included in the xsd and the mapping file.
+Once selected the root entity a diagram of the entities available will appear. Uncheck the attributes that should not be included in the XSD and the mapping file.
 
 
 Meteo Stations example 
 ----------------------
 
-To show in details how to configure the ``DataStore`` we will use the Meteo Stations use case.
+This section provides an example of configuring the ``DataStore`` with weather station information.
 
-.. warning:: This is an oversimplified use case.
+.. note:: This is a relatively simple use case, and more complicated scenarios may require more work.
```

Comment:
This note was meant to inform a possible user belonging to some information community (as usually AppSchema users are) that the use case used in the documentation is a simplified one in the sense that is not referring to any know use case that should conform to some information model, to avoid confusion due the possible existance of metereological stations models.

Reply:
Will have another go at that. The focus should be that example shown is not intended to be representative of any real schema or standard?

Reply:
Correct, while the module can be used to reach conformity to a given target application schema, it addresses the needs of those that want a complex information structure. Think of how we manage simple features today, by simply reflecting the database structure: it does not make them the golden standard of interoperability, but there's plenty of practical usage outside of it.

Reply:
Finally got a chance to go back to this. Any additional suggestions would be appreciate.